lib/propolis/src/firmware/smbios/table.rs

```diff
@@ -381,6 +381,22 @@ impl Table for Type1 {
     }
 }
 
+impl Default for Type1 {
+    fn default() -> Self {
+        Type1 {
+            manufacturer: SmbString::default(),
+            product_name: SmbString::default(),
+            version: SmbString::try_from(crate::version())
+                .expect("version string should not contain NULs"),
```

Comment:
If we remove the bhyve API version from the string, we could include this condition as part of the `version()` unit tests